src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketScmFormFillDelegate.java

```diff
@@ -178,4 +197,9 @@ public ListBoxModel doFillServerIdItems(String serverId) {
     public boolean getShowGitToolOptions() {
         return false;
     }
+
+    private BitbucketMirrorHandler createMirrorHandlerUsingRepoSearch() {
```

Comment:
/nit not entirely sure why this needs its own method when it is only called once

Reply:
because descriptor has all of these dependencies and i don't want to have setters in BitbucketSCM.